dbFunc.js

```diff
@@ -0,0 +1,101 @@
+const { Db } = require("mongodb");
+const DataStorage = require("./data.js");
+require("dotenv").config();
+
+//my DB URL
+const url = `mongodb+srv://emilysaber:${process.env.DBPASS}@cluster0.yjuvt.mongodb.net/`;
+//****** EM FIGURE OUT HOW TO GET ENV PW IN HERE BEFORE SUBMITTING!!!!! */
```

Comment:
Great comment!

Comments aren't always used to explain what is going on through your code. It is also a great way to communicate with other people on your projects